java-vertexai/google-cloud-vertexai/src/main/java/com/google/cloud/vertexai/generativeai/GenerativeModel.java

```diff
@@ -76,14 +79,15 @@ public GenerativeModel(String modelName, VertexAI vertexAi) {
    *     that will be used by default for generating response
    * @param tools a list of {@link com.google.cloud.vertexai.api.Tool} instances that can be used by
    *     the model as auxiliary tools to generate content.
-   * @param vertexAI a {@link com.google.cloud.vertexai.VertexAI} that contains the default configs
+   * @param vertexAi a {@link com.google.cloud.vertexai.VertexAI} that contains the default configs
    *     for the generative model
    */
   private GenerativeModel(
       String modelName,
       GenerationConfig generationConfig,
       ImmutableList<SafetySetting> safetySettings,
       ImmutableList<Tool> tools,
+      Optional<Content> systemInstruction,
```

Comment:
Can we add a `setSystemInstruction` in the GenerativeModel.Builder class? Also a getter method in the GenerativeModel